crates/db_schema/src/schema.rs

```diff
@@ -299,6 +303,17 @@ diesel::table! {
     }
 }
 
+diesel::table! {
+    federation_queue_state (id) {
+        id -> Int4,
+        #[max_length = 255]
+        domain -> Varchar,
```

Comment:
Might be easier to use `instance_id` here?

Reply:
I guess it would be equivalent, so it's mostly a stylistic choice. I used domain so the table itself is simpler to read and understand without any joins

Reply:
I agree that instance_id would be much better, for normalization purposes.